docs/getting-started/plugins.md

```diff
@@ -12,46 +12,48 @@ Amplication uses _Plugins_ to extend the functionality of your generated code.
 
 Some plugins can be enabled when you first create your service.
 For example, if you choose MySQL as your database, the _MySQL DB_ plugin will be enabled.
-If you choose to turn on authentication for your service, that will enable the _NestJS Auth Module_ plugin.
+If you choose to turn on authentication for your service, that will enable the _NestJS Auth Module_ plugin and other Authentication-focused plugins.
 
-All community plugins published by Amplication are available on the _All Plugins_ page.
-Your installed plugins are shown on the _Installed Plugins_ page.
-Plugins can be manually enabled or disabled on either of these pages.
+All community plugins published by Amplication are available on your service's _Plugins_ tab.
+Plugins can be enabled or disabled on this tab.
+To view your installed plugins, click on the _Installed Plugins_ category in the filter options on the left side.
```

Comment:
I'm not sure the "installed plugins" in a Category- as you can see the title of Categories **below** the "installed plugins" option. Maybe we should keep the "page" terminology for the "installed plugins" option. WDYT?
Note that this comment is relevant in other places of this PR as well